trace-telemetry-exporter/README.md

```diff
@@ -0,0 +1,19 @@
+# trace-telemetry-exporter
+
+A tool which uses the OpenMetrics format to record the following:
```

Comment:
Could this be formatted in a tabular format, similar to https://github.com/charlie-haley/omada_exporter?tab=readme-ov-file#-metrics